README.md

```diff
@@ -71,7 +71,11 @@ to install `protractor`and use the following `configs.js` file to run e2e tests:
 'use strict';
 (function(){
   module.exports = {
-    url     : 'http://192.168.100.200/openattic/#/login',
+    urls: {
+      base: 'http://192.168.100.200',
+      ui: '/openattic/#/',
```

Comment:
For Vagrant boxes the UI URL is /#/

Reply:
I'm just changing the format, and keeping the same value that we have before.

This is the configuration example that should be used when we run openattic inside the docker container (with apache), so we should use "/openattic" to run the tests.

(this configuration was tested on my environment)